revocation/crl/cache/bundle.go

```diff
@@ -0,0 +1,199 @@
+// Copyright The Notary Project Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cache
+
+import (
+	"archive/tar"
+	"crypto/x509"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"time"
+)
+
+const (
+	// BaseCRL is the file name of the base CRL
+	PathBaseCRL = "base.crl"
+
+	// Metadata is the file name of the metadata
+	PathMetadata = "metadata.json"
+)
+
+// Bundle is in memory representation of the Bundle tarball, including base CRL
+// file and metadata file, which may be cached in the file system or other
+// storage
+//
+// TODO: consider adding DeltaCRL field in the future
+type Bundle struct {
+	BaseCRL  *x509.RevocationList
+	Metadata Metadata
+}
+
+// Metadata stores the metadata infomation of the CRL
+//
+// TODO: consider adding DeltaCRL field in the future
+type Metadata struct {
+	BaseCRL  FileInfo  `json:"base.crl"`
+	CreateAt time.Time `json:"createAt"`
```

Comment:
Request doc for the meaning of `CreateAt`.

Reply:
Updated.